configs/metrics/run.sh

```diff
@@ -4,7 +4,7 @@ set -e -x
 snakemake \
   --profile .profiles/local \
   --configfile \
-    configs/metrics/example_workflow.yaml \
+    configs/metrics/hlcav1.yaml \
```

Comment:
```suggestion
    configs/metrics/example_workflow.yaml \
```

consider creating your own runner script in a diffrerent instead of editing the example runner script